docs/custom-pipelines.rst

```diff
@@ -249,5 +251,54 @@ the entry point to the pipeline under the ``[options.entry_points]`` section.
         pipeline_name = scancodeio_scan_and_report_pipeline.pipelines.scan_and_report:ScanAndReport
 
 .. tip::
-    Take a look at `Google LicenseClassifier pipeline for Scancode.io <https://github.com/nexB/scancode.io-pipeline-glc_scan>`_
+    Take a look at `Google License Classifier pipeline for Scancode.io <https://github.com/nexB/scancode.io-pipeline-glc_scan>`_
     for a complete example on packaging a custom tool as a pipeline.
+
+Publishing Pipeline to PyPI
+---------------------------
+After successfully packaging a pipeline, one may want to distribute it as a plugin via PyPI.
+Ensure that you have a directory structure similar to :ref:`pipeline_packaging_example`, and all
+the package files are correctly configured.
+
+.. tip::
+    See the `Python packaging tutorial at PyPA <https://packaging.python.org/tutorials/packaging-projects/>`_
+    for a detailed setup guide.
+
+Next step involves generating the distribution archives for the package. Make sure you have the
+latest version of
+``build`` installed on your system
+
+.. code-block:: bash
+
+    pip install --upgrade build
+
+Now run the following command from the same directory where :guilabel:`pyproject.toml` is located:
+
+.. code-block:: bash
+
+    python -m build
+
+Once completed, you should have two files present in the ``dist`` directory with extensions ``.tar.gz`` and ``.whl``.
```

Comment:
Once completed, you should have two files inside the ``dist`` directory with the ``.tar.gz`` and ``.whl`` extensions.

.. note:: 
    Remember to create an account on `PyPI <https://pypi.org/>`_ before uploading your distribution archive to PyPI.